packages/create-plugin/templates/common/playwright.config

```diff
@@ -0,0 +1,52 @@
+import { defineConfig, devices } from '@playwright/test';
+import { dirname } from 'node:path';
+
+const pluginE2eAuth = `${dirname(require.resolve('@grafana/plugin-e2e'))}/auth`;
+
+/**
+ * Read environment variables from file.
+ * https://github.com/motdotla/dotenv
+ */
+// require('dotenv').config();
+
+/**
+ * See https://playwright.dev/docs/test-configuration.
+ */
+export default defineConfig({
```

Comment:
```suggestion
export default defineConfig<PluginOptions>({
```

Reply:
Thanks for spotting this - should be done now. 👍